sql/database.sql

```diff
@@ -0,0 +1,39 @@
+-- Item class
+
+CREATE TABLE item (
+	id INT NOT NULL GENERATED ALWAYS AS IDENTITY,
+	genre_id INT,
+	author_id INT,
+	label_id INT,
+	source_id INT,
+	publish_date DATE,
+	archived BOOLEAN,
+	FOREIGN Key(genre_id) REFERENCES genre(id),
+	FOREIGN Key(author_id) REFERENCES author(id),
+	FOREIGN Key(label_id) REFERENCES label(id),
+	FOREIGN Key(source_id) REFERENCES source(id),
+	PRIMARY KEY(id)
+);
+
+-- Create indexes --
+
+CREATE INDEX idx_genre_id ON item (genre_id)
+CREATE INDEX idx_author_id ON item (author_id)
+CREATE INDEX idx_label_id ON item (label_id)
+CREATE INDEX idx_source_id ON item (source_id)
+
+-- Author and Game classes
+
+CREATE TABLE author (
+	id INT GENERATED BY DEFAULT AS IDENTITY,
+	first_name VARCHAR(100),
+	last_name VARCHAR(100),
+	items TEXT [],
```

Comment:
- [ ] Please note that you can't translate every detail from the UML diagram into the schema file, as implementing arrays for the `items` here makes sense in the UML diagram and in code but not in the schema file. Please fix this.

Reply:
@ShadyShawkat
thanks for your good and valuable feedback, done